addons/hr_timesheet/views/rating_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="rating_rating_view_search_project_inherited" model="ir.ui.view">
+        <field name="name">rating.rating.search.project.inherited</field>
+        <field name="model">rating.rating</field>
+        <field name="inherit_id" ref="project.rating_rating_view_search_project"/>
+        <field name="arch" type="xml">
+            <xpath expr="//filter[@name='my_ratings']" position="after">
+                <filter string="My Team's Ratings" name="my_team_rating" domain="[('rated_partner_id.user_ids.employee_parent_id.user_id', '=', uid)]"/>
+                <filter string="My Department's Ratings" name="my_department_rating" domain="[('rated_partner_id.user_ids.employee_id.member_of_department', '=', True)]"/>
```

Comment:
`employee_ids` will not give the same result as` user_ids.employee_id` because `employee_ids` field in res.partner is related to `address_home_id` field not `user_partner_id`.